tribe-ext-recurring-cleanup/index.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * Plugin Name:     The Events Calendar PRO Extension: Cleanup recurring events
+ * Description:     Adds a recurring event cleanup tool to WP Admin > Tools > Available tools.
+ * Version:         1.0.0
+ * Extension Class: Tribe__Extension__Recurring_Cleanup
+ * Author:          Modern Tribe, Inc.
+ * Author URI:      http://m.tri.be/1971
+ * License:         GPLv2 or later
+ * License URI:     https://www.gnu.org/licenses/gpl-2.0.html
+ */
+defined( 'WPINC' ) or die;
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) { die( '-1' ); }
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) { return; }
```

Comment:
For both `ABSPATH` check and `Tribe__Extension` check, would encourage:
• Breaking these into multiple lines instead of one-liners
• Adding space between the two. So, instead of this once expanded into multiple lines:
```
}
// Do not load unless Tribe Common is fully loaded.
```
This:
```
}

// Do not load unless Tribe Common is fully loaded.
```